packages/babel-parser/README.md

```diff
@@ -18,9 +18,9 @@ thanks to the awesome work of [@RReverser](https://github.com/RReverser) and [@m
 
 ## API
 
-### `babylon.parse(code, [options])`
+### `babel.parse(code, [options])`
```

Comment:
Since these references are in the Babylon readme itself, this documentation is specifically about the `.parse` function on the parser package, not the `.parse` function on the `babel-core` package. In this case we should document these as `babelParser.parse`.